boards/arm64/imx8mm_evk/dummy.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CONFIG_UART_MCUX_IUART
+
+#define DT_DRV_COMPAT nxp_imx_iuart
+
+#include <device.h>
+#include <drivers/uart.h>
+#include <drivers/clock_control.h>
+#include <errno.h>
+#include <soc.h>
+
+static int dummy_init(const struct device *dev)
+{
+	return 0;
+}
+
+#define IUART_MCUX_INIT(n)						\
+									\
+	DEVICE_DT_INST_DEFINE(n,					\
+			    &dummy_init,				\
+			    NULL,					\
+			    NULL,					\
+			    NULL,					\
+			    PRE_KERNEL_1,				\
+			    CONFIG_SERIAL_INIT_PRIORITY,		\
+			    NULL);
+
+DT_INST_FOREACH_STATUS_OKAY(IUART_MCUX_INIT)
+
+#endif
```

Comment:
what's going on here?

Reply:
This file is not needed after rebase, dropped.

Reply:
Seems CI not happy if this file dropped. Because UART is not enabled for now, so CI report.
samples/subsys/shell/shell_module/src/uart_reinit.c:19: undefined reference to `__device_dts_ord_9'

Bring it back